tools/spec-gen-sdk/src/automation/workflowPackage.ts

```diff
@@ -117,13 +117,15 @@ const workflowPkgDetectArtifacts = async (context: WorkflowContext, pkg: Package
  * Copy sdk artifact path to {work_dir}/out/stagedArtifacts
  */
 const workflowPkgSaveSDKArtifact = async (context: WorkflowContext, pkg: PackageData) => {
+  const language = pkg.language ?? getLanguageByRepoName(context.sdkRepoConfig.mainRepository.name);
   const relativeFolderPathParts = pkg.relativeFolderPath.split('/');
-  const serviceName = relativeFolderPathParts[relativeFolderPathParts.indexOf('sdk') + 1];
+  let serviceName = relativeFolderPathParts[relativeFolderPathParts.indexOf('sdk') + 1];
+  if (language.toLocaleLowerCase() === 'go') {
```

Comment:
Language is English only.
```suggestion
  if (language.toLowerCase() === 'go') {
```